stats/handlers.go

```diff
@@ -0,0 +1,23 @@
+package stats
+
+import "context"
+
+// RPCTagInfo defines the relevant information needed by RPC context tagger.
+type RPCTagInfo struct {
+	// PeerID
+	PeerID string
+	// PeerName
+	PeerName string
+	// FullMethodName is the RPC method in the format of /package.service/method.
+	FullMethodName string
+}
+
+// Handler defines the interface for the related stats handling (e.g., RPCs, connections).
+type Handler interface {
+	// TagRPC can attach some information to the given context.
+	// The context used for the rest lifetime of the RPC will be derived from
+	// the returned context.
+	TagRPC(context.Context, *RPCTagInfo) context.Context
+	// HandleRPC processes the RPC stats.
+	HandleRPC(context.Context, RPCStats)
```

Comment:
I don't know if `HandleRPC` is the best naming for this. It's not handling the RPC, is just seemingly processing RPC stats. `LogEvent`, `LogStats` perhaps better?

Reply:
I would like to keep it as close to the `grpc` api that inspired this as it makes it easier to understand how the code has changed between the `grpc` package and this package. And also it makes porting the OpenCensus grpc plugin to gorpc easier to grok.